packages/wonder-blocks-form/src/components/text-field.tsx

```diff
@@ -233,12 +237,10 @@ class TextField extends React.Component<PropsWithForwardRef, State> {
             // Should only include Aria related props
             ...otherProps
         } = this.props;
+
         return (
-            <input
-                // @ts-expect-error: we shouldn't be passing `style` to `css()`
-                // here b/c `style` allows nested arrays of styles, but `css()`
-                // only allows a flat array.
-                className={css([
+            <StyledInput
+                style={[
```

Comment:
@n1colemejia I moved the `style` prop up to where the `className` prop was.  This makes it easier to see what was changed in the array.